scripts/acn/Dockerfile.dev

```diff
@@ -0,0 +1,35 @@
+FROM ubuntu:18.04
```

Comment:
useful to deploy the public dht from current branch (mostly develop right before release)